src/libraries/System.Drawing.Common/pkg/buildTransitive/System.Drawing.Common.targets

```diff
@@ -0,0 +1,7 @@
+<Project InitialTargets="_ValidateSystemDrawingCommonNotNetCoreApp2x">
+  <Target Name="_ValidateSystemDrawingCommonNotNetCoreApp2x"
+          Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $(TargetFrameworkVersion.StartsWith('v2.'))">
+    <Error Condition="'$(SuppressTfmSupportBuildWarnings)' == ''"
+           Code="SYSLIB9001" Text="System.Drawing.Common doesn't support $(TargetFramework). Consider updating your TargetFramework to netcoreapp3.0 or later." />
```

Comment:
Do we need to add this `SYSLIB` code to the md file where we list them all?

https://github.com/dotnet/runtime/blob/207b03a073107e63df0b67056dd596226065651e/docs/project/list-of-diagnostics.md

Reply:
wow cool, didn't know this exists. We also missed documenting System.Speech: https://github.com/dotnet/runtime/blob/6af59187b5a74988a97bc0b0a13742682e30519b/src/libraries/System.Speech/pkg/build/System.Speech.targets#L5.